ducktape/services/service.py

```diff
@@ -26,13 +26,14 @@
 # should be large enough to use one instance per service instance.
 
 from ducktape.command_line.config import ConsoleConfig
+from ducktape.template import TemplateRenderer
 
-
-class Service(object):
-    def __init__(self, service_context):
+class Service(TemplateRenderer):
+    def __init__(self, service_context, *args, **kwargs):
         """
         :type service_context: ServiceContext
         """
+        super(Service, self).__init__(*args, **kwargs)
```

Comment:
Thanks for the links.

Very minor comment, since I don't have a strong opinion here - perhaps just be consistent wrt to super call between Service and Test (Service __init__ method makes call to super, but Test.__init__ doesn't)